frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -818,7 +844,13 @@ pub mod pallet {
 					}
 				},
 				_ => T::WeightInfo::on_initialize_nothing(),
+			};
+
+			// increment the number of election blocks for this round if not in `Phase::Off` block.
+			if current_phase.is_signed() || current_phase.is_unsigned() {
+				ElectionBlocksCount::<T>::mutate(|c| c.saturating_inc());
```

Comment:
Do we need this? Can this not be calculated by keeping the block number when the round starts (transitions from phase off to phase signed) and calculating the difference, i.e. `Current block` - `Block when current round started`?

Reply:
I think that counting the number of electing blocks is more flexible and allow us to have the emergency throttling spanning through many rounds. E.g. we can set `MinElectingBlocks = (SignedPhase + UnsignedPhase) * 2`, which would allow up to 2x `fn elect()` to fail per era.